lewis/adapters/stream.py

```diff
@@ -78,13 +78,29 @@ def __init__(self, host, port, target):
         self.bind((host, port))
         self.listen(5)
 
+        self._accepted_connections = []
+
     def handle_accept(self):
         pair = self.accept()
         if pair is not None:
             sock, addr = pair
             print("Client connect from %s" % repr(addr))
             StreamHandler(sock, self.target)
 
+            self._accepted_connections.append(sock)
```

Comment:
`sock`s are appended to `accepted_connections` but only removed on server shutdown. What about connections closed by the client?